pkg/cmdlets/checkout/checkout.go

```diff
@@ -103,12 +103,23 @@ func (r *Checkout) Run(ns *project.Namespaced, branchName, cachePath, targetPath
 		}
 	}
 
+	// Match the case of the organization.
+	// Otherwise the incorrect case will be written to the project file.
+	owners, err := model.FetchOrganizationsByIDs([]strfmt.UUID{pj.OrganizationID})
+	if err != nil {
+		return "", errs.Wrap(err, "Unable to get the project's org")
+	}
+	if len(owners) == 0 {
+		return "", errs.New("Unable to get the project's org")
```

Comment:
This needs localization, as this will likely get quite a few hits (eg. misstyped org name).

It should also be an input error.

Reply:
If the user mistypes an org name, an error will be triggered all the way back on line 62 (`model.FetchProjectByName()` failed). If we get to this point, the org exists, it's just that the lookup by UUID failed for whatever reason (I cannot see that happening). Regardless, I will localize this error to be safe.